dell/latitude/9430/default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
```

Comment:
I copied these imports from the 7430 config. There were some more settings in 7430 which I didn't copy over because I don't know what they do. To be honest I don't know what `intel`, `laptop` and `ssd` do neither, but they sound generic enough that they should be applied here ;)

Reply:
If you have a specific intel generation, you can also include it: https://github.com/NixOS/nixos-hardware/tree/master/common/cpu/intel

Reply:
SSD mainly enables fstrim.

Reply:
The generation is Alder Lake which currently does not exist below `cpu/intel` - do you think it's best to create it, add the kernel param there and then import it here?